### PR TITLE
vb-1024, additional support requests using journey loop

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
## Description
Additional Support tests now use the journey loop, to run through both the book and amend journeys

### CODE REVIEW

File: server/routes/visitJourney/additionalSupport.test.ts [hunk 1]
```diff
@@ -46,379 +52,385 @@ afterEach(() => {
...
+          expect($('[data-test="WHEELCHAIR"]').prop('checked')).toBe(true)
+          expect($('[data-test="OTHER"]').prop('checked')).toBe(true)
+          expect($('#otherSupportDetails-error').text()).toContain('Enter details of the request')
+          expect(flashProvider).toHaveBeenCalledWith('errors')
+          expect(flashProvider).toHaveBeenCalledWith('formValues')
+          expect(flashProvider).toHaveBeenCalledTimes(2)
+        })
+    })
   })
 })
```
Comment (psoleckimoj) [inline server/routes/visitJourney/additionalSupport.test.ts:233]: Not sure you need to end thee loop and restart on 235? Could just remove 233 and 235? Not tried but should work.

**Comment by tpmcgowan** [comment]
I think tests have failed again because of the update journey feature flag issue.

**Comment by hutcheonb-moj** [comment]
remembered the flag as soon as I changed it to a full PR, working on it now, thanks!

File: server/routes/visitJourney/additionalSupport.test.ts (hunk 1 above)
Comment (hutcheonb-moj) [reply server/routes/visitJourney/additionalSupport.test.ts:233]: I had tried it without and found the grouping was strange, will re-attempt though while I add the flag

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**